src/features/application-search/ApplicationSearchBar.tsx

```diff
@@ -137,7 +139,8 @@ const ApplicationSearchBar = ({ fontSize }: { fontSize?: string }) => {
 
     // Navigate to search page then set shared state to initiate a search
     if (!onSearchPage && !navigatedToSearch && search) {
-      navigate(Routes.Routes().search.fullPath).then((_v) => setSearch(search));
```

Comment:
react router navigate function does not allow the use of .next() after navigate call